docs/.markdownlint.json

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
If we're adding any link files to the repo, we should add the corresponding github action to enforce it

imo let's either remove this file from the PR, or add a github actions file to run the markdown lint check

Reply:
^feel free to disagree if you have a strong opinion on this

Reply:
Name: markdownlint
Id: DavidAnson.vscode-markdownlint
VS Marketplace Link: https://marketplace.visualstudio.com/items?itemName=DavidAnson.vscode-markdownlint

Personally I'd prefer just use this and let my IDE tell me what I should and shouldn't be doing - until there is a more well defined structure for markdown within the project.

Because it might just slow people down in the meantime, while those rules are being tweaked.

Either way, could merge it in now and then the action later, or keep this open and wait for an action.

I'm moving house tomorrow, so might be a few days before I get a chance to write up that action, although you're welcome to write the action yourself if you want, of course.

Reply:
Makes sense! I'll just merge this in now, and add the github action later.